DataFormats/L1THGCal/src/classes_def.xml

```diff
@@ -42,26 +43,32 @@
   <class name="edm::Wrapper<l1t::HGCalClusterBxCollection>"/>
 
   <class name="l1t::HGCalClusterT<l1t::HGCalCluster>" />
-  <class name="l1t::HGCalMulticluster" ClassVersion="14">
+  <class name="l1t::HGCalMulticluster" ClassVersion="16">
+   <version ClassVersion="16" checksum="1276395758"/>
+   <version ClassVersion="15" checksum="777879934"/>
```

Comment:
I'm wondering, if people (in particular in the L1T group) have been reprocessing samples and produce version 15 objects and still want to read them, it could be useful to keep track of version 15?

Reply:
In that case it can be kept, but in general I would not recommend producing samples for long-term use with private versions of data formats...